### PR TITLE
Swerve Input

### PR DESCRIPTION
**Pull request by tcdoan**
Added input class to Swerve, and debugged order of lists in swerve modules in order for the swerve rotation to work. Swerve now works, and input scales to SwerveConstant speed constants.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Input.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.XboxController;
+import frc.robot.constants.JoystickConstants;
+
+/** Add your docs here. */
```
Comment (geeknoid) [inline src/main/java/frc/robot/subsystems/Input.java:11]: How about writing a description?

**Review by TejaKoripellaUwU — COMMENTED** [review]
nice. It works physically and the comments are good.

**Review by TejaKoripellaUwU — APPROVED** [review]
It seems like the code works on the robot. Comments are good. Nice job

**Review by TejaKoripellaUwU — APPROVED** [review]
Looks good, maybe clean up the pathplanner folder so that we don't have too many unnecessary paths.

**Pull request merged**